### PR TITLE
feat: Add total commits to [GitHubCommitActivity]

### PR DESCRIPTION
**Pull request by jNullj**
As part of a new feature proposed at issue #6070 added the requested feature. I also used the conversation at pull request #6081 as a basis for those changes.

This change adds a new interval to the github/commit-activity shield 'total' (t for short). The interval shows the total commits of the repo since its creation.

photo for fun
image

### CODE REVIEW

**Comment by jNullj** [comment]
A few min, i will run Prettier...

**Comment by jNullj** [comment]
I am changing the label to "commits" when using total interval, wait for the next commit...

**Comment by calebcartwright** [comment]
It's a little unclear what the current status of this PR is. If you're still in the process of working on things, it's best to mark the PR as a draft so that it's explicitly clear to us it's still a WIP and we don't need to invest much time in looking at it yet.

If you are done and ready for a review, it'd helpful if you could indicate such :+1:

**Comment by jNullj** [comment]
Oh, my bad, i am ready for a review.
Is there a dedicated way to indicate the PR is ready for review on github?

**Review by chris48s — APPROVED** [review]

**Comment by chris48s** [comment]
on board with this solution :+1:

File: services/github/github-commit-activity.tester.js [hunk 1]
```diff
@@ -11,6 +12,11 @@ const isCommitActivity = Joi.alternatives().try(
   isZeroOverTimePeriod
 )
 
+t.create('commit acticity (total)').get('/t/badges/shields.json').expectBadge({
```
Comment (msberends) [inline services/github/github-commit-activity.tester.js:15]: Typo here, but not sure if it breaks functionality?

Comment (jNullj) [reply services/github/github-commit-activity.tester.js:15]: My bad, what bugs me is that CI was not able to catch it, I will take a look at it later.

Comment (chris48s) [reply services/github/github-commit-activity.tester.js:15]: Yes it is just a test name, and I already merged a fix in https://github.com/badges/shields/commit/64d3380e1f83a7c2c9d05a80577980c2bcab1f31 no more follow-up needed

**Comment by calebcartwright** [comment]
> Oh, my bad, i am ready for a review. Is there a dedicated way to indicate the PR is ready for review on github?

Sorry I forgot to respond to this, but no worries, and yes you can convert a [draft PR to a "ready" state](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#marking-a-pull-request-as-ready-for-review) as well as [mark a "ready" PR as still being in draft-mode](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft)

**Pull request merged**